cypress/e2e/hermioneAccount.spec.cy.js

```diff
@@ -0,0 +1,66 @@
+import { faker } from '@faker-js/faker';
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  const depositAmount = `${faker.number.int({ min: 500, max: 1000 })}`;
+  const withdrawAmount = `${faker.number.int({ min: 50, max: 500 })}`;
+  const balance = depositAmount - withdrawAmount + 5096;
+  const user = 'Hermoine Granger';
+  const firstAccountNumber = '1001';
+  const secondAccountNumber = '1002';
+  const currentBalance = '5096';
+  const totalBalance = (+depositAmount) + (+currentBalance);
```

Comment:
The calculation of `totalBalance` is correct because you are using the unary plus operator to convert strings to numbers. Ensure consistency by applying similar conversions where needed.